lib/renderer/renderer-mewn.go

```diff
@@ -5,7 +5,7 @@ package renderer
66f62d378693593309352586536533b5aa996058c07bee797156641e5b6cda9cdbe5f948faf1b8f9389da89713c60deca3df9cddea1268878fe3b6065527993d6a692ec825b9d85316f61b9d032de67a510bd88f138676a92cc17fd65bb03b4f07f4b1b4e23425572acf236f2372612e48e4f516b2c8eebc883ec1bf7750fa68f6731c910bcd3081598207c8a3af37fc55992c8738e50e500462cb1d94367f84d885c4bb0382849c23fc8832a6cdfa566d918899f22a7e02322a846621553edd5481b4a6b439882ad0c88a21aeeb330d7aa4a466a6201795d4747607dca9b15a53f293b3fb129ce8f26e08dff275db3afb0c0b7bd9703c7587c2dba1be62b48d44d4be68e3fff2f68373ea40a79d669f2d146bf09f94c9ecf61f58319039f3c9e225497e37997c3f7df7eeed5fbffbfebbc9bb77d3371f95df0857ada52c54c9bfeda43932b4e2d6e859f91c511369537a65522c44c098df38bb8fd0e1cf87023a3a18637d84a8452a4a735596912a2315b51b1216a8dfe8921bc63db2f572caf1b7b86ac4e5ac2ab54d945045911fea3a6618bf9c8e24527e22a5a4fe524e59087c739626776d9a6ce66621f1afd369be68aa013abdad6e9ae08c2876e586ba0196d778b8a2041ec1609536aff36ebe1065aed3d645e5d6bb2d185fb27a984e190b7526df9f39f8804720c9db3c867df44b7b7582f6e4c2303e9af494ae2d47d3d03b7ad5214dafda664495a55e9bd36918c85e35a709fcd0393954cf8a815e7673735824ed3287dd8667ff53b1f6dc31ecb4b0cbf5d80987ae3b604d902a02f4910a0db7117e30993c06def40f33a3fd4b656b16181f961439fc713a1d037fb66505fd5779bc85fd4fb598c4cffb9ff942a4d6a4ca5323ca22d79e124158f55943c9bd3cdf19c512daea34edaa939393c4fdd04e5c4e137721a7f529f50300d78d1ea8656c807f9344a475c27212e932c28c52d1a3ca75167d558faa4c9d2e7ca433305eaf343841184bfcdc2e50fce776218fa1caaeb95d84baba83286c415973aafeafa7eaffef54bd904f640c4bf3e051d432abcb948a263dfc4927c3867b94dfae87ec6bcf992a8034c890f50bb3bd6404ee036581b7baf21cedc4c80c529bc1974fb32bbb2dac0183c03b28729502fdf65fe5c5b76b4e08eb87e693cb77ea72b538be0d38f5e73f8dab20b8c31164250c857225b62c4dd6427b7932337528d12a2a5459428675b3eda9e2a88b3f88a617e144441f9bb62422aca3264b6aeddd515fd7f91a415fcb8c93758471579d712b8b966e6ded97a40d4af44d77aae3e49b0a73076b5d7ac846a311694f3a2fa67a782ad6dc1c946b9b042bceaa3be319e4e021422f78b3c37b2bea924f9b0116e34855fb2908945916f8adf57a75889f952294ec5ed17c25ee8dd8325e275e35d468db30f3701cfda8da5dcc5c86d0e9217430802ea3e7b0a1635109c62372a6724f44d7582322ac1091ac343bd8ae9851cdc663dfea87e35316aa7ae443e01fb2ecbe4aa92737ac2f95d9b44a8fe669799d03fea2a4ce42044678f8c34bc33bcba5cd0e28b060b2ab8dce3364078cc7d7556b343318efabfbfb1754efd5b3fc2107c212c044fce0bdd3cb9d074a50fa092778feb7695912c64154b6f71b00ff7ef843a465f9b9f2338633df9e9c8936b73683ea71de4f6e4065a753f7730dbe71aefce9f05955ed20256844b0ff616727d4b7d63e9e61a867bf5ddd20ad0db8b308e8baf8637fc86fe5f1c6ae63c79bde2db6bc02b78c8f77e6094a77cda3f4ce7cdce55e1739c477fc7aab7d7ccd7f05e5fc12d439b2fd7b0adb98e41e7930d0ad2a8966c6837b5479f55045f572d87f77a30373ec5042e5c187f4c1d87d0ed91ae2270fb1111e3268d4b2ae4153fda26867f0db449bd6ed812a184ebe4185c38d28c6e8f7f8eba08999e14186b27075f6e8e3b75d79b6a64af0f3ff8ef2bc7a880c97881bbb164d257af3e6cd9be84aedd61b1f7d319b4a47b25a6e229c23e7c57fb8b4974b6c9e5eb3faf2e9062de0758b1b6d20bab568e55fb7bab2f96e6b1a3bf7ba5d1502b4b12cf06b7c19eabc8cf1f90c196161c192ff040000ffff8bf286145a140000")
```

Comment:
Is there a reason why the wails.js hash data has changed?

Reply:
Must be from commits before my changes in this branch since I used the original branch as a base.

Reply:
I'll drop it out of this branch.

Reply:
No need. Looks like there wee dependency updates in npm.